src/Benchmarks/link-a-thon.api-template.json

```diff
@@ -74,5 +74,8 @@
     "Source": {
       "Project": "src/GrpcService/GrpcService.csproj"
     }
+  },
+  "LinkAThonTrimmedAndR2RSingleFileWithTrimListCrossgen2": {
```

Comment:
I'll update the zip and try this out before merging.

Reply:
Try to create names that are more concise. Like `LinkAThonTrimmedR2RSingleCrossgen2`

But technically it doesn't matter so it's just a question of preference.